scripts/install_slice_db.pl

```diff
@@ -68,3 +78,15 @@
     exit 1;
 }
 
+sub db_exists {
+    my $db_name = shift;
+    my $user = $GK_DB_USER;
+    my $pass = $GK_DB_PASS;
+
+    no autodie qw/system/;
+    my $return_value = system("mysql -u $user -p$pass -e \"use $db_name\" > /dev/null 2>&1");
```

Comment:
Just wondering: is there a Perl API that could do this, or is a system call the best way to do it?

Reply:
I think it's probably the simplest way to do it.  I looked a little bit at Perl APIs for this like DBI and they are fairly complicated to use and not core modules.